sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateTableAsSelect.scala

```diff
@@ -91,6 +93,6 @@ case class CreateTableAsSelect(
   }
 
   override def argString: String = {
-    s"[Database:$database, TableName: $tableName, InsertIntoHiveTable]\n" + query.toString
+    s"[Database:$database, TableName: $tableName, InsertIntoHiveTable]"
```

Comment:
@chenghao-intel I feel this string is better than having a string containing `tableDesc` (I feel it has way too much information for a user).